services-camel/src/main/java/edu/unc/lib/cdr/routing/MetaServicesRouter.java

```diff
@@ -42,38 +42,43 @@ public void configure() throws Exception {
         from("{{fcrepo.stream}}")
             .routeId("CdrMetaServicesRouter")
             .to("direct-vm:index.start")
-            .to("direct:process.ingest");
-
-        from("direct:process.ingest")
-            .routeId("ProcessIngest")
-            .filter(simple("${headers[org.fcrepo.jms.eventType]} contains 'ResourceCreation'"
-                    + " && ${headers[org.fcrepo.jms.identifier]} regex '.*(original_file|techmd_fits)'"
-                    + " && ${headers[org.fcrepo.jms.resourceType]} contains '" + Binary.getURI() + "'"))
+            .to("direct:process.enhancement");
 
+        from("direct:process.enhancement")
+            .routeId("ProcessEnhancement")
+            .filter(simple("${headers[org.fcrepo.jms.eventType]} contains 'ResourceCreation'"))
                 // Trigger binary processing after an asynchronously
                 .threads(enhancementThreads, enhancementThreads, "CdrEnhancementThread")
                 .delay(simple("{{cdr.enhancement.postIndexingDelay}}"))
-                .to("direct:process.creation");
-
-        from("direct:process.creation")
-            .routeId("ProcessCreationEvent")
-            .log(LoggingLevel.DEBUG, "Processing binary metadata for ${headers[org.fcrepo.jms.identifier]}")
-            .removeHeaders("CamelHttp*")
-            .to("fcrepo:{{fcrepo.baseUrl}}?preferInclude=ServerManaged&accept=text/turtle")
-            .process(mdProcessor)
-            .choice()
-                .when(simple("${headers[org.fcrepo.jms.identifier]} regex '.*original_file'"))
-                    .to("direct-vm:replication")
-                    .to("direct:process.enhancements")
-                .when(simple("${headers[org.fcrepo.jms.identifier]} regex '.*techmd_fits'"))
-                    .to("direct-vm:replication")
-                .otherwise()
-                    .log(LoggingLevel.WARN, "Cannot process binary metadata for ${headers[org.fcrepo.jms.identifier]}")
-            .end();
+                .removeHeaders("CamelHttp*")
+                .to("fcrepo:{{fcrepo.baseUrl}}?preferInclude=ServerManaged&accept=text/turtle")
+                .process(mdProcessor)
+                .multicast()
+                .to("direct:process.binary.original", "direct:process.solr");
 
+        from("direct:process.binary.original")
```

Comment:
Since this processes original_file|techmd_fits it might be best to rename the route to just "process.binary"